### ISSUE TITLE
Promotional-Who different color ticks

### ISSUE THREAD

**Issue by Tayzarcarr** [Tayzarcarr/d1095#7]
Promotional-Who different color ticks
![image](https://cloud.githubusercontent.com/assets/13252937/15612998/271de500-2429-11e6-90e6-9073ba2afdf3.png)

**Comment by billyfrost** [comment]
Yes, this is so the ticks stand out on the green bg - think this is fine
